ci-operator/step-registry/gcp/provision/private-dns-zone/gcp-provision-private-dns-zone-ref.yaml

```diff
@@ -0,0 +1,24 @@
+ref:
+  as: gcp-provision-private-dns-zone
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  grace_period: 10m
+  commands: gcp-provision-private-dns-zone-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: BASE_DOMAIN
+    default: ""
+    documentation: The base domain.
+  - name: PRE_CREATE_CLUSTER_PVTZ
```

Comment:
I think we can remove `PRE_CREATE_CLUSTER_PVTZ`, generally once some workflow call `gcp-provision-private-dns-zone`, that means the workflow want to pre-create some private cluster zone, but not need an extra var to control the behavior.